free-programming-books-it.md

```diff
@@ -68,8 +68,8 @@
 
 * [Guida a Java 8](http://twiki.di.uniroma1.it/pub/Metod_prog/RS_INFO/lezioni.html)
 * [Introduzione a Java](http://www.ateneonline.it/hyperbook/j_book/java2.htm)
-* [Java - Wikibooks](https://it.wikibooks.org/wiki/Java)
-* [Object Oriented && Java 5 (II Edizione) - Claudio De Sio Cesari](http://www.claudiodesio.com/download/oo_&&_java_5.zip)
+* [Java](https://it.wikibooks.org/wiki/Java) - Wikibooks
+* [Object Oriented && Java 5 (II Edizione)](http://www.claudiodesio.com/download/oo_&&_java_5.zip) - Claudio De Sio Cesari (ZIP)
```

Comment:
Oops, I really was not careful enough. My comment about `ruby-it` being only available as a zip was supposed to go here, not on the ruby one. This book exists only as a zip. I'm still looking for a zip alternative for the ruby-it.org one.